apps/web/src/services/auth/password/validation-email/ValidationEmailPage.vue

```diff
@@ -97,12 +97,12 @@ watch(() => route.query.status, (res) => {
                 <div v-if="state.userId"
                      class="collapsible-wrapper"
                 >
-                    <p-collapsible-toggle v-if="!state.isCollapsed"
+                    <p-collapsible-toggle v-if="state.isCollapsed"
                                           v-model="state.isCollapsed"
```

Comment:
nitpick: 
```suggestion
                                          v-model="state.is-collapsed"
```

`Kebab` case is rule!